02-front-end/11bloco-react-intro/01dia-intro/exercicio-11-1/src/Content.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react'
+
+const conteudos = [
+  {
+    conteudo: 'High Order Functions',
+    bloco: 8,
+    status: 'Aprendido'
+  },
+  {
+    conteudo: 'Composicao de Componentes',
+    bloco: 11,
+    status: 'Aprendendo',
+  },
+  {
+    conteudo: 'Composicao de Estados',
+    bloco: 12,
+    status: 'Aprenderei'
+  },
+  {
+    conteudo: 'Redux',
+    bloco: 16,
+    status: 'Aprenderei'
+  },
+];
+
+export default class Content extends Component {
+  render() {
+    return (
+      <section className='card-section'>
+        {conteudos.map((tarefa, index) => (
+          <div
+            key= {index}
```

Comment:
Key na verdade era pra ter o valor da chave conteúdo do objeto tarefa, o enunciado não foi nada descritivo 😅 mas era isso que ele queria:
```suggestion
            key= {tarefa.conteudo}
```